### DISCUSSION TITLE
support udp?

### DISCUSSION THREAD

**Discussion by malikshi** [Qv2ray/gun#17] [Q&A] [answered]
I see new release support for UDP transport, so basic support for gaming or video call right?

**Comment by DuckSoft** [comment, marked as answer]
Yes. The UDP support was done by @studentmain. Now the tunnel can tunnel UDP content as well.
Still, we lack testers and developers. If you find any glitch, feel free to report to us.

**Reply by malikshi** [reply]
![Screenshot_1071](https://user-images.githubusercontent.com/9080737/107017842-44ccd600-67d2-11eb-9fb1-30d416a59385.jpg)
I try it with Netch and give me NAT 1 which is can be used for games or any UDP support. but it's more often getting log like the picture above. 
`;loglevel=none` adding to plugin_opts make it broken with your plugin.

**Reply by DuckSoft** [reply]
Correct. We haven't supported any log level control. So expect those warnings.

**Reply by malikshi** [reply]
I'm not using fiber for my connection, so I can't provide you the speed performance. Anyway, thank you for creating this Qv2ray. I hope in the future to release SIP003 APK for Shadowsocks android.
